apps/api/src/app/shared/helpers/liquid.ts

```diff
@@ -20,6 +20,6 @@ const stringifyDataStructureWithSingleQuotes = (value: unknown, spaces: number =
 
     return valueEscapedNewLines;
   } else {
-    return String(value);
+    return value == null ? '' : String(value as unknown);
```

Comment:
I like this shortcut, but because this is very sensitive part i wonder if we should go with the more explicit version "value === null || value === undefined" instead. 
What do you think?

Reply:
What's different with that version ?

Reply:
String(value) when value is undefined returns `undefined`. Isn't that the reason we started this PR in the first place?